bycycle/plts/features.py

```diff
@@ -0,0 +1,129 @@
+"""Plot cycle features."""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from neurodsp.plts.utils import savefig
+
+###################################################################################################
+###################################################################################################
+
+@savefig
+def plot_feature_hist(feature, burst_param, bins='auto', ax=None, **kwargs):
+    """Plot a histogram for a cycle feature.
+
+    Parameters
+    ----------
+    feature : pandas.DataFrame or 1d array
+        Dataframe output from :func:`~.compute_features` or a 1d array.
+    burst_param : str
+        Column name of the parameter of interest in ``df``.
+    bins : int or string, optional, default: 'auto'
+        The number of bins or binning strategy string,
+        as specified in matplotlib.pyplot.hist.
+    ax : matplotlib.Axes, optional, default: None
+        Figure axes upon which to plot.
+    **kwargs
+        Keyword arguments to pass into `matplotlib.pyplot.hist`.
+
+    Notes
+    -----
+    Default keyword arguments include:
+
+    - ``xlabel``: str, default: ``burst_param``
+    - ``figsize``: tuple of (float, float), default: (10, 10)
+    - ``color``: str, default: 'k'
+    - ``xlim``: tuple of (float, float), default: None
+    - ``fontsize``: float, default: 15
+    - ``alpha``: float, deafult: .5
+
+    """
+
+    # Limit dataframe to bursts
+    if isinstance(feature, pd.core.frame.DataFrame):
+        feature = feature[feature['is_burst']][burst_param]
+
+    # Optional keyword args
+    figsize = kwargs.pop('figsize', (5, 5))
+    color = kwargs.pop('color', 'k')
+    xlabel = kwargs.pop('xlabel', burst_param)
+    xlim = kwargs.pop('xlim', None)
+    fontsize = kwargs.pop('fontsize', 15)
+    alpha = kwargs.pop('alpha', .5)
+
+    if ax is None:
+        fig, ax = plt.subplots(figsize=figsize)
+
+    # Axis style
+    ax.set_xlabel(xlabel, size=fontsize)
+    ax.set_ylabel('# cycles', size=fontsize)
+
+    if xlim:
+        ax.set_xlim(xlim)
+
+    ax.hist(feature, bins=bins, color=color, alpha=alpha, **kwargs)
+
+
+@savefig
+def plot_feature_catplot(df, burst_param, group_by=None, ax=None, **kwargs):
```

Comment:
I have a pedantic nitpick with the name `catplot`. It comes from seaborn, AFAIK, but I'm not sure it's such a general name. Also, since it means 'categorical plot', this name reads as 'plot feature categorical plot', which annoys me. 

I'm not so sure if there is a better name - but maybe something more general / agnostic, like `plot_feature_comparison`, or (since I think one could plot this without multiple categories) `plot_feature_data` or similar.

Reply:
What about `plot_feature_categorical`, since it plots a feature given a categorical variable to group by? Otherwise, I like `plot_feature_comparison` over `plot_feature_data`.

Reply:
I would vote for `plot_feature_comparison` over `plot_feature_categorical`. Although these are reasonably called 'categorical' plots, I think I'd say the 'category' is basically the feature, right? (as in, features are the 'categories' we have in our data). I'm happy with `plot_feature_comparison`

Reply:
The feature is the continuous variable, not the categorical. The category is a group (patient vs control, etc) that a single feature is compared between. `plot_feature_comparison` kinda sounds like two features are going to be compared.

Reply:
Ohhh right. Yeh, of course, you're totally right. 'Categorical' makes sense - but also this plot doesn't actually necessitate a category or comparison, right? As in `groupby` can be None, and it just plots on feature, no comparison across categories. Damnit, why is naming things so hard! I don't even know anymore, I'm going to cede this decision to you :) Call it whatever you think is best!